src/runtime.js

```diff
@@ -22,6 +27,15 @@ let tornado = {
   },
 
   /**
+   * Register a helper, overwrite if a helper already exists
+   * @param {String} name The name of the helper
+   * @param {Function} method The function to be executed when the helper is found in a template
+   */
+  registerHelper(name, method) {
```

Comment:
I don't think it matters if the helpers object is public or private, so long as there is only one object that stores the helpers, because if there is only one object, you blow away previously helpers.

Are you suggesting that we have an internal "Tornado Helpers" object and a separate "Client Helpers" object, where a helper with the same name can be registered on both?